packages/core/package.json

```diff
@@ -56,6 +56,7 @@
     "dev": "modern build --watch"
   },
   "dependencies": {
+    "on-finished": "2.4.1",
```

Comment:
We do not want to add any 3rd dependency to the core package. It is recommended to implement It by ourselves. The compromise solution is prebundle